test/e2e-tests.sh

```diff
@@ -66,6 +119,14 @@ go_test_e2e -timeout=20m ./test || fail_test
 # Run these _after_ the integration tests b/c they don't quite work all the way
 # and they cause a lot of noise in the logs, making it harder to debug integration
 # test failures.
-./examples/smoke-test.sh || fail_test
-
+header "Running YAML e2e tests"
+if ! run_yaml_tests; then
+  failed=1
+  echo "ERROR: one or more YAML tests failed"
+  # If formatting fails for any reason, use yaml as a fall back.
+  kubectl get pipelinerun.pipeline.knative.dev -o=custom-columns-file=./test/columns.txt || \
```

Comment:
This checks for status of pipelinerun only and not taskrun. I think testing for taskrun as well would be good. 
I would prefer the following order
- check for taskrun status to be successful 
- check for pipelinerun status to be successful

We can add examples of one off task and taskruns too in example in future.